<launch>

 <!--include file="$(find sbpl_arm_planner_node)/sbpl_arm_planner_node.launch"/-->

 <param name="sbpl_exec/arm_trajectory_topic_name" value="/trajectory_controller/trajectory_command"/>
 <param name="sbpl_exec/arm_trajectory_service_start" value="/trajectory_controller/TrajectoryStart"/>
 <param name="sbpl_exec/arm_trajectory_service_query" value="/trajectory_controller/TrajectoryQuery"/>
 <param name="sbpl_exec/sbpl_planner_service_name" value="/sbpl_planner/plan_path/GetPlan"/>
 <param name="sbpl_exec/effort_command" value="/effort_controller/set_command"/>

 <param name="sbpl_exec/shoulder_pan_goal" value="0"/>
 <param name="sbpl_exec/shoulder_pitch_goal" value="-0.4"/>
 <param name="sbpl_exec/upperarm_roll_goal" value="0"/>
 <param name="sbpl_exec/elbow_lift_goal" value=".05"/>
 <param name="sbpl_exec/forearm_roll_goal" value="0"/>
 <param name="sbpl_exec/wrist_lift_goal" value="0"/>
 <param name="sbpl_exec/wrist_roll_goal" value="0"/>

 <!-- Executive -->
  <node pkg="sbpl_arm_executive" type="sbpl_arm_executive" name="sbpl_exec" output="screen"/>

</launch> 